src/internal/transactionSubmission.ts

```diff
@@ -160,3 +167,37 @@ export async function submitTransaction(args: {
   });
   return data;
 }
+
+export async function signAndSubmitTransaction(args: {
+  aptosConfig: AptosConfig;
+  signer: Account;
```

Comment:
I like the idea of using `Account` here. Similar concept to where I would like to replace many `address: HexInput` to `address: AccountAddress` :)

Reply:
I think we use `Account` for all transactions submission api functions